src/neo4j_graphrag/experimental/pipeline/kg_builder.py

```diff
@@ -81,7 +81,7 @@ class SimpleKGPipeline:
         text_splitter (Optional[Any]): A text splitter component. Defaults to FixedSizeSplitter().
         pdf_loader (Optional[Any]): A PDF loader component. Defaults to PdfLoader().
         kg_writer (Optional[Any]): A knowledge graph writer component. Defaults to Neo4jWriter().
-        on_error (str): Error handling strategy. Defaults to "RAISE". Possible values: "RAISE" or "IGNORE".
+        on_error (str): Error handling strategy. Defaults to "CONTINUE". Possible values: "RAISE" or "CONTINUE".
         perform_entity_resolution (bool): Merge entities with same label and name. Default: True
         text_splitter (Optional[Any]): A text splitter component. Defaults to FixedSizeSplitter().
```

Comment:
This line is duplicated.

Reply:
Also, it seems I forgot to add `perform_entity_resolution` in the docstring, would you mind adding it here? Or I can create a new card for that.

Reply:
`perform_entity_resolution` is currently there. I removed the duplicate line

Reply:
My bad, thanks!